src/fixes.rs

```diff
@@ -30,18 +30,26 @@ use log::debug;
 use lsp_types::Url;
 use salsa::InternKey;
 
-use crate::context::{get_fix_for_diagnostic_message, get_name_for_fix_message};
+use crate::context::get_fix_for_diagnostic_message;
 use crate::db::FixerDatabase;
 use cairo_lang_defs::db::DefsGroup;
 use cairo_lang_semantic::db::SemanticGroup;
 use cairo_lang_syntax::node::db::SyntaxGroup;
 
-/// Represents a fix for a diagnostic, containing the span of code to be replaced,
-/// the suggested replacement, and a short description of the fix.
+/// Represents a suggestion for a fix, containing the span of code to be replaced,
+/// and the suggested code to replace it with.
 #[derive(Debug, Clone, Eq, PartialEq, Hash)]
-pub struct Fix {
+pub struct Suggestion {
     pub span: TextSpan,
-    pub suggestion: String,
+    pub code: String,
+}
+
+/// Represents a fix for a diagnostic, containing the span of diagnosed code,
+/// the suggested replacements, and a short description of the fix.
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct Fix {
```

Comment:
The name is a bit weird - it's not a fix but rather a set of suggestions with assigned diagnostic + location